framework/decode/metadata_consumer_base.h

```diff
@@ -34,6 +34,9 @@ GFXRECON_BEGIN_NAMESPACE(decode)
 /// implement to handle metacommands.
 class MetadataConsumerBase
 {
+  protected:
+    uint64_t block_index_;
```

Comment:
nitpicking: I think we tend to declare private variable members at the bottom of the class declarations (after private function declarations)